src/test/groovy/com/commercehub/gradle/plugin/avro/FunctionalSpec.groovy

```diff
@@ -72,7 +72,7 @@ abstract class FunctionalSpec extends Specification {
     }
 
     protected void addDefaultRepository() {
-        buildFile << "repositories { jcenter() }\n"
+        buildFile << "repositories { jcenter()\n}\n"
```

Comment:
Is there a reason a newline is needed here?

Reply:
No this is a mistake, will fix